docs/pages/bare/overview.mdx

```diff
@@ -25,52 +26,183 @@ If you've ever written a native module for React Native, you'll be surprised how
   href="/bare/upgrade/"
 />
+
+### New workflows
+
+Once your app has `expo` package installed, you can submit your app to app stores with a single command or update configure `expo-updates` library to manage remote updates to you app's code:
+
+<BoxLink
+  title="App distribution"
+  Icon={BuildArtifactIcon}
+  description="Build and submit your app to app stores with a single command."
+  href="/distribution/introduction"
+/>
+
+<BoxLink
+  title="Install expo-updates"
+  Icon={BuildArtifactIcon}
+  description={
+    <>
+      Learn how to install and configure <CODE>expo-updates</CODE> to manage remote updates and
+      enable PR previews.
+    </>
+  }
+  href="/bare/installing-updates/"
+/>
+
+### New mindset
+
+The following helps with your project's long term maintainability, native code maintenance, and easier upgrades:
+
 <BoxLink
   title="Adopt Prebuild"
   Icon={RefreshCw02Icon}
   description="Learn how to simplify maintaining your native projects by generating them on demand from configuration."
-  href="/guides/adopting-prebuild"
+  href="/guides/adopting-prebuild/"
 />
+
 <BoxLink
-  title="App distribution"
-  Icon={BuildArtifactIcon}
-  description="Build and submit your app to the app store with a single command."
-  href="/distribution/introduction"
+  title="Expo Router"
+  Icon={RouterLogo}
+  description="Expo Router is a file-based routing library that offers advantages such as organized navigation hierarchy, automatic deep linking support, and more."
+  href="/router/introduction/"
 />
+
+## Common questions
+
+<Collapsible summary="How long will it take to adopt Expo in my existing React Native project?">
+
+Adopting Expo doesn't have to be done in one step. You can start with _quick wins_ and then move to tackle more complex parts. You can also pick and choose which features you want to adopt at the beginning of this process.
+
+</Collapsible>
+
+<Collapsible summary="What will I gain from using Expo in my React Native App?">
+
+Adopting Expo tools in your existing React Native app will help you maintain long-term maintenance support, easier native upgrades, and a better developer experience.
+
+</Collapsible>
+
+<Collapsible summary="Who uses Expo?">
+
+Expo is used by top companies worldwide that serve millions of end users. For more information, see our [Expo showcase](https://expo.dev/customers).
+
+</Collapsible>
+
+<Collapsible summary="What impact will adopting Expo have on my app's size?">
+
+The `expo` package has a small footprint since it only includes a minimal set of modules required in every app with autolinking infrastructure and other Expo SDK libraries that are built-in.
+
+Including `expo` package in your app adds 1 MB to the final size of apps on app stores. This package has a marginal cost. For example, on Android, it only adds 150 Kib and the rest of the size comes from the language runtime (Kotlin).
```

Comment:
I think we have an FYI page about this we could link to

Reply:
Happy to link but I'm not sure which one. There are multiple pages:
- Android APK specific one (although it doesn't target production apps): https://github.com/expo/fyi/blob/main/android-app-size.md
- Manage app size/prebuild: https://github.com/expo/fyi/blob/main/managed-app-size.md

Reply:
Maybe this is the benchmark (since it was updated 3 months ago):

![CleanShot 2024-10-05 at 01 42 03@2x](https://github.com/user-attachments/assets/b768a8dc-79bc-43d0-80e7-caeb2d01dd88)

Reply:
Pretty brief, but this is probably the link of record at the moment: https://docs.expo.dev/faq/#what-is-the-minimal-size-of-a-hello-world-expo-app

Reply:
Since the new guide on App size is now published, I'll adjust the above context and include the link.